CvGameCoreDLLUtil/include/CvEnums.h

```diff
@@ -15,6 +15,16 @@ enum CLOSED_ENUM eCiv5GameCoreMPoolType
 	c_eCiv5GameplayDLL = c_eMPoolTypeUserStart + 0x100,		// Adding 0x100 because we don't want to collide with the app's IDs
 };
 
+#if defined(MOD_EXE_HACKING)
```

Comment:
this will always be true, you need to use regular if statement if you want to do runtime check